README.md

```diff
@@ -95,6 +95,7 @@ echo $select->sql();
 - `orderBy(array ...pairs)` either `[column]` or `[column, direction]`
 - `limit(int limit)`
 - `offset(int offset)`
+- `distinct(bool or null)` defaults to `false`
```

Comment:
Fixed...defaults to *false*. Should really have my coffee before I try to code...